data/transactions/logic/eval_test.go

```diff
@@ -1226,7 +1226,10 @@ const globalV9TestProgram = globalV8TestProgram + `
 `
 
 const globalV10TestProgram = globalV9TestProgram + `
-// No new globals in v10
+global AssetCreateMinBalance
+global AssetOptInMinBalance
+>=
+&&
```

Comment:
minor, but could you have this code verify the values returned by the opcodes?

I see the v1 test program does:

```
global MinBalance
int 1001
==
```

Reply:
done